lib/bosh/gen/generators/new_release_generator/templates/templates/make_manifest.tt

```diff
@@ -59,7 +59,11 @@ perl -pi -e "s/PLACEHOLDER-DIRECTOR-UUID/$DIRECTOR_UUID/g" $tmpdir/stub-with-uui
 perl -pi -e "s/NAME/$NAME/g" $tmpdir/stub-with-uuid.yml
 perl -pi -e "s/STEMCELL/$STEMCELL/g" $tmpdir/stub-with-uuid.yml
 
-spiff merge \
+if ! [ -x "$(command -v spruce)" ]; then
```

Comment:
ok, thanks; but move this validation to the top of the script, before the expensive `bosh` calls, so we get fast failure

I tested the PR; worked. Merge when you're ready.